src/modules/todoUtils.js

```diff
@@ -0,0 +1,40 @@
+import { getTodosFromStorage, updateTodosInStorage } from './storage.js';
+
+export const completeTodo = (description) => {
+  const todos = getTodosFromStorage();
+  const todo = todos.find((todo) => todo.description === description);
+
+  // update todo completed state
+  todo.completed = !todo.completed;
+
+  const index = todos.indexOf(todo);
+  todos[index] = todo;
+
+  updateTodosInStorage(todos);
+};
+
+export const removeTodo = (description) => {
```

Comment:
- [ ] Good job, but when I deleted a task , It removed from Local Storage but I have to refresh the browser to see him deseapear from the app . Please fix it in order to make the deleting function works as expected on both dom and local storage 👍 

https://www.loom.com/share/3d0ab39d293545ce84f2f5f9ecf2a1ae?sid=c18dccdb-ad8f-4022-a51f-4a6029393a8d

Reply:
Hi @codecaiine. You're right. Thank you for your review. Let me fix it